homeassistant/components/bluetooth/passive_update_processor.py

```diff
@@ -52,7 +52,9 @@ class PassiveBluetoothDataUpdate(Generic[_T]):
     )
 
 
-class PassiveBluetoothProcessorCoordinator(BasePassiveBluetoothCoordinator):
+class PassiveBluetoothProcessorCoordinator(
```

Comment:
Could you subclass the coordinator and overload async_handle_bluetooth_event instead?

Reply:
Hmm, I think this behaviour makes sense as the default, which is why I didn't do it in a subclass. If you disagree I am happy to do it in a subclass instead. But i'd be submitting an `ActiveBluetoothProcessorCoordinator` seperately for xiaomi, and I think this behaviour especially makes sense for any passive sensor that has binary sensor and sensor payloads in the same broadcast, and any passive sensor that has dynamic platforms.

Reply:
I think its ok to change it, but we need to label it a breaking change since its shipped this way in 2022.8

Reply:
Ah ok my bad - didn't think of API stability.

Reply:
Actually, if i revert [c8e0421](https://github.com/home-assistant/core/pull/76384/commits/c8e0421edef33c8a5c7e95c15e4bcc8144e40992) it won't be a breaking change.